General coding practices/Test Automation (Mobile) - element IDs.md

```diff
@@ -0,0 +1,98 @@
+**Chapters:**
```

Comment:
Removed ✅ 
I added it initially so that mobile devs can easily find and jump to the paragraphs for their respective platforms - iOS or Android (XML vs Compose). So that they don't have to read through the whole text in order to find the section for their platform - because not all of it is relevant to them, just the sections for their respective platform.
It's a quick update, I can re-add it if you think it makes sense to have it 😄